bip-0432.mediawiki

```diff
@@ -0,0 +1,141 @@
+
+===Example Use Cases===
+
+* Batched payments with segregated fee pools: Batched payments that can be fee bumped without access to customer deposit-related key material
+* Simplified watchtowers/accelerators: No requirement to equip watchtowers with privileged key material, and no value to steal by those watchtowers
+* Multiparty smart contracts: LN BOLTs can be revamped to remove value-sapping anchors.
+* LN-Symmetry: During the contest period, no anchor value can be sourced endogenously.
+* [https://bitcoin.stackexchange.com/questions/100537/what-is-spacechain Spacechains]
+* Ark transactions, for anchors and connector outputs
+* Timeout Trees, for anchors and connector outputs
+
+===Related Work===
+
+[https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-January/021334.html SIGHASH_GROUP] style proposals are an alternative method of bringing funds to a transaction without involving CPFP by enacting a softfork. Making these pin-resistant may require follow-on policy work, or [https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-May/020458.html more general covenants] to directly stop pins we want to avoid. The drawback of these are the necessity of a softfork.
+
+[https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2020-September/018168.html Transaction Sponsors] is a softfork proposal to allow transactions to
+sponsor transactions with no explicit relationship in the classical UTXO
+model. Ephemeral Dust with an un-keyed anchor can be viewed as a type of opt-in transaction sponsors implemented
+purely in policy.
+
+Using a 0-value CPFP anchors is not a new idea, see:
+
+* [https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2018-May/015931.html LN-dev discussion on 0-value anchors]
+
+The discussion lacked a solution to the issue of the dust entering into the UTXO set
+causing negative externalities.
+
+==Definitions==
+
+Ephemeral dust: An output with dust value which is immediately spent by a child transaction.
+
+Ephemeral dust transaction: A transaction that has an ephemeral dust output.
+
+==Specification==
+
+When received by a peer for inclusion to the mempool an ephemeral dust transaction MUST:
+
+* Be an otherwise valid transaction
+* Be only considered for mining with ephemeral dust spent
+* Have only one ephemeral dust output (output values which would normally cause rejection)
+
+or will be rejected by policy. All other policy checks are left in place.
+These constraints apply only to mempool policy. Blocks are not
+invalidated by breaking these policy-only rules as they have no bearing on consensus.
+
+==Rationale==
+
+This policy is a relaxation to dust policies that most node software enforces.
+
+If a dust output never ends up unspent in a mining template, then the marginal
+exposure of the network to dust is minimized. Most identified use-cases
+of ephemeral dust only require a single dust output, so multiple dust outputs
+are left for a possible future extension.
```

Comment:
```suggestion
If dust outputs never ends up unspent in a mining template, then the marginal
exposure of the network to dust is minimized.
```

Deleting since the "aspirational" spec isn't limited to a single dust output.

Reply:
Removing the single-anchor mention.

As for the other parts, you could imagine a linear chain of dust being spent(each tx 0-fee), with the ultimate transaction bringing exogenous fees to pay for the chain, so I'm not sure being specific about feerates of children is helpful for aspirational section.

In cluster mempool terms, the implementation could allow dust as long as it's all created and spent intra-chunk(or would be trimmed), since the miner template would never select a block that would result in the dust entering the UTXO set. It may not be worth the complexity, but seems possible at least, and much more general than current implementation.